### PR TITLE
Allow to configure timeoutSeconds in livenessProbe and readinessProbe

### PR DESCRIPTION
**Pull request by peter-englmaier**
Introduce new values to configure timeoutSeconds in livenessProbe/readinessProbe:

.database.internal.livenessProbe.timeoutSeconds
.database.internal.readinessProbe.timeoutSeconds

Default values (1 second) unchanged if parameters are not set.

In GKE the internal database reports "Readiness probe failed". Setting .database.internal.readinessProbe.timeoutSeconds: 3
fixes the problem.

Please let me know if I need to add documentation for the new values elsewhere.

### CODE REVIEW

**Comment by peter-englmaier** [comment]
I have tested the helm chart in gke. After setting timeout to 3 seconds, our instance does no longer get 'readiness probe failed' errors:
image

File: values.yaml [hunk 1]
```diff
@@ -777,6 +777,12 @@ database:
     #  requests:
     #    memory: 256Mi
     #    cpu: 100m
+    # The timeout used in livenessProbe; 1 to 5 seconds
+    livenessProbe:
```
Comment (ywk253100) [inline values.yaml:781]: Could you add the same configuration for all other components so that they are consistent?

Comment (peter-englmaier) [reply values.yaml:783]: What other components? I want to touch the internal database only.

**Comment by peter-englmaier** [comment]
> @peter-englmaier could you please update the README.md file to add items of
> 
> ```
> .database.internal.livenessProbe.timeoutSeconds
> .database.internal.readinessProbe.timeoutSeconds
> ```
>
> and then rebase the latest master into your working branch?

OK, done. 
There was a [comment](https://github.com/goharbor/harbor-helm/pull/1307#discussion_r1000302140) which is open.

**Review by zyyw — APPROVED** [review]

File: values.yaml (hunk 1 above)
Comment (peter-englmaier) [reply values.yaml:783]: @ywk253100 : what other components to you have in mind? Yes, the probes have more parameters and as a regular harbor user, I also like to have things be consistent.

Comment (peter-englmaier) [reply values.yaml:783]: @ywk253100 : Can you please comment what you need for your approval? I'm a bit lost. I can add more configurable parameters, but which ones? My PR is for liveness/readinessProbes for the build-in database only. I can also use other parameter names if you find them not suitable. Best, Peter

**Review by ywk253100 — APPROVED** [review]

**Comment by peter-englmaier** [comment]
Thanks @zyyw !

**Pull request merged**